lib/infer/return.js

```diff
@@ -17,7 +17,14 @@ module.exports = function () {
     if (comment.returns) {
       return comment;
     }
-    var fn = finders.findTarget(comment.context.ast);
+    var fn = findTarget(comment.context.ast);
+
+    // In case of `/** */ var x = function () {}` findTarget returns
+    // the declarator.
+    if (t.isVariableDeclarator(fn)) {
```

Comment:
I feel like my change to findTarget might have been bad. However, my other PR makes findTarget return a Path and then it would work to have it return the init again (because paths allows getting the parent).